INSTALL.md

```diff
@@ -50,6 +58,9 @@ Other Fedora versions or distributions may require some adaptation.
    git clone https://github.com/candlepin/subscription-manager.git
    cd subscription-manager/
    ```
+   
+   ***NOTE**: For `./setup.py build_ext --inplace` to succeed, it is necessary to really clone the `subscription-manager` repository and not just upload the files. Otherwise, the command would fail on `git describe`.*
+
```

Comment:
Why would you do that? :)

This note should not be necessary, you just have to follow the instructions.

Reply:
The scenario is you have the repository cloned on the desktop you use to write code and uploaded to a (virtual) machine with RHEL to actually run the software or its test.

Reply:
Oh, okay. Still, I believe you should configure your VM to mount the directories from the host, so you don't have to synchronize it.

While your use-case is valid, it won't scale well :)